archetypes/pattern/cluster-sizing.adoc

```diff
@@ -0,0 +1,15 @@
+---
+title: Cluster sizing
+weight: 20
+---
+
+:toc:
+:imagesdir: /images
+:_content-type: ASSEMBLY
+
+// Add asciidoc modules here
```

Comment:
Can we add a suggestion about using the [include::](https://docs.asciidoctor.org/asciidoc/latest/directives/include/) directive to add modules?